DS & Algo/src/Week1/RemoveLinkedListElement.java

```diff
@@ -0,0 +1,40 @@
+package Week1;
+
+public class RemoveLinkedListElement {
+
+    public ListNode removeElements(ListNode head, int val) {
+
+        //Check if head not equals null
+        if(head!=null){
+            if(head.val==val){
+                //moving head to next till the value is equal to the target
+                while(head!=null && head.val==val){
+                    head = head.next;
+                }
+            }
```

Comment:
no need for if(head.val==val)  as you are checking this condition in while block too. this will avoid nesting and will make code clean